doc/developer/dircache.md

```diff
@@ -401,17 +401,31 @@ it's not doing it at all.
 
 **By default nothing changes, as the default value for `dircache validation freq = 1`.**
 
-To try this speed up, configure it;
+To try this speed up, configure it:
 
 ```ini
 dircache validation freq = 1-100
 dircache metadata window = 60-3600
 dircache metadata threshold = 10-1800
+dircache files = yes/no  # New in 4.x: Allow file caching
```

Comment:
nitpick: all these four options are "New in 4.x" so it's a bit misleading to call out just this one.

besides, like my comment from before, how about we remove this whole section too from the developer docs? they're arguably more of user docs appropriate for the Configure chapter.